README.md

```diff
@@ -210,3 +210,18 @@ with open("pages.txt", mode="wt", encoding="utf-8") as fp:
 
 logging.info("pages.txt file created")
 ```
+
+## Reading dumps from local files
+
+You can also read dumps from local, non-compressed XML files:
+
+```pythom
+from mediawiki_dump.dumps import LocalFileDump
+from mediawiki_dump.reader import DumpReader
+
+dump = LocalFileDump(dump_file="test/fixtures/dump.xml")
+reader = DumpReader()
+
+pages = [entry.title for entry in reader.read(dump)]
+print(dump, pages)
```

Comment:
@matkoniecz

Reply:
Thanks! This works fine!

(and maybe it previous failed due to bz2 vs gz confusion? On looking again my unpacking code was confused)

I will need to figure out how to read just latest page version without processing deleted pages or outdated versions, but that is a separate thing.

Thanks again!

Reply:
Great, no worries :)